go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -752,7 +757,7 @@ func (bnc *BaseNetworkController) addLogicalPortToNetwork(pod *kapi.Pod, nadName
 func (bnc *BaseNetworkController) updatePodAnnotationWithRetry(origPod *kapi.Pod, podInfo *util.PodAnnotation, nadName string) error {
 	resultErr := retry.RetryOnConflict(util.OvnConflictBackoff, func() error {
 		// Informer cache should not be mutated, so get a copy of the object
-		pod, err := bnc.watchFactory.GetPod(origPod.Namespace, origPod.Name)
+		pod, err := bnc.kube.GetPod(origPod.Namespace, origPod.Name)
```

Comment:
Removed, I was dealing with a race now everythign use the watrchFactory and there is as little recheck seting annotation to fix unit test since retry on conflict do not work there.

Reply:
0_o still see it

Reply:
My bad fixing